docs/contributor/telemetry-load-test/assets/log-fluentbit-max-pipeline.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: telemetry.kyma-project.io/v1alpha1
+kind: LogPipeline
+metadata:
+  name: load-test-2
+spec:
+  output:
+    http:
+      dedot: false
```

Comment:
Let's enable de-dotting. Even though, it's not required by the backend running a Lua script is something with a potential performance impact and where we might have regressions.

Reply:
dedot enabled